### ISSUE TITLE
[BITV] User menu set status button

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#41022]
The "Set status" entry is currently an `a` tag but should be a `button`

![image](https://github.com/nextcloud/server/assets/24800714/cdf7efcb-a7a2-442f-bb70-08efa20d8221)
